services/user.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
The import of `User` from `db.models` is unnecessary since `get_user_model()` is used. Consider removing this import.

Reply:
I do this for annotation of the type of return value functions